docs/DafnyRef/Attributes.md

```diff
@@ -328,7 +310,7 @@ the functionality is already adequately described where
 refinement is described.
 -->
 
-### 22.1.14. tailrecursion
+### 22.1.12. `{:tailrecursion true/false}`
 This attribute is used on method declarations. It has a boolean argument.
```

Comment:
It can also be used with no arguments, in which the case the argument defaults to `true` (like it does for most/all other attributes that take a boolean argument).